src/matching/Matching_generic.ml

```diff
@@ -74,7 +74,13 @@ let logger = Logging.get_logger [ __MODULE__ ]
 (* incoming environment *)
 type tin = {
   mv : Metavariable.bindings;
-  stmts_match_span : Stmts_match_span.t;
+  (* This is for storing the subset of statements that are actually
+   * matched in Match_patterns.match_sts_sts.
```

Comment:
The comment doesn't explain why we need to keep them.

Reply:
We could also say here that the statements are kept in "reversed" order.

Reply:
yep